src/i18n/strings/en_EN.json

```diff
@@ -28,5 +28,7 @@
     "Decentralised, encrypted chat &amp; collaboration powered by $matrixLogo": "Decentralised, encrypted chat &amp; collaboration powered by $matrixLogo",
     "Sign In": "Sign In",
     "Create Account": "Create Account",
-    "Explore rooms": "Explore rooms"
+    "Explore rooms": "Explore rooms",
+    "Anyone can knock on this room to join. You can change this at any time from room settings.": "Anyone can knock on this room to join. You can change this at any time from room settings.",
+    "Knocking": "Knocking"
```

Comment:
The changes to the language files don't look right to me - they should only be changed using the `yarn i18n` script. Since there are no new strings being added to the code here, the files should not change